clarity/src/vm/ast/parser/v2/lexer/mod.rs

```diff
@@ -1386,13 +1428,14 @@ mod tests {
             LexerError::InvalidCharPrincipal('O')
         );
 
+        /* cylewitruk: Commenting out this reserved placeholder, assuming it might have been reserved for Bitwise NOT anyway?
```

Comment:
I don't think it was reserved for that.  You'll have to ask @obycode.  It might be for Clarinet or something.

Reply:
This test is just testing that `~` is detected as an unknown symbol. The `Placeholder` is used as a replacement token so that parsing can continue past the problematic symbol. With your changes in place to use `~` as a valid symbol, instead of removing this test, you could instead use a different invalid character here, `@` for example.

Reply:
Gotcha - this has been fixed now :)